history/characters/60000_63000_lordaeronian.txt

```diff
@@ -4558,15 +4561,41 @@
 	culture=azerothian religion=holy_light # Tan skin
 	martial=6 diplomacy=6 stewardship=5 intrigue=4 learning=5
 	father=60686 #Bolvar
-	597.6.19={ birth=yes trait=creature_human }
-	597.6.19={
-		culture=lordaeronian
+	597.6.19={ 
+		birth=yes trait=creature_human culture=lordaeronian
+		effect = { make_important_lore_character_effect = yes }
 	}
 	603.3.1={
 		employer=48080 #Daelin
+		trait=journaller	#Writes to Bolvar
 	}
 	631.5.20={ death=yes }
 }
+#Mara Fordragon, a lore character
+mara_fordragon={
```

Comment:
Don't let her be in dynasty without conections to other members. It may lead to weird dynasty line.
Connect her to Bolvar somehow: his grand aunt, etc.

Reply:
Extended Fordragon family tree to permit Mara as Bolvar's paternal aunt.